webapp/utils/format.ts

```diff
@@ -1,14 +1,17 @@
 import { type Account } from 'btc-wallet/unisat'
 import { shorten } from 'crypto-shortener'
 import { smartRound } from 'smart-round'
-import { type Address } from 'viem'
+import { type Address, type Hash } from 'viem'
 
 export const formatBtcAddress = (account: Account) =>
   shorten(account, { length: 5 })
 
 export const formatEvmAddress = (address: Address) =>
   shorten(address, { length: 4, prefixes: ['0x'] })
 
+export const formatTxHash = (txHash: Hash) =>
```

Comment:
I think we were using it to format Bitcoin transactions as well

```suggestion
export const formatTxHash = (txHash: BtcTransaction | Hash) =>
```

(then, the casting where this function was used can be removed)